app/models/miq_worker.rb

```diff
@@ -345,9 +345,32 @@ def start_runner
     Process.detach(pid)
     pid
   end
+  alias_method :start_runner_via_fork, :start_runner
```

Comment:
ok, I'm not sure what happened to that commit, let me follow the bouncing ball...

* the worker was added in https://github.com/ManageIQ/manageiq/pull/13551
* it was changed to not use a heartbeating thread during the 10 minute ansible setup in https://github.com/ManageIQ/manageiq/pull/14053/
* the thread knowledge was moved out of the worker/runner and to the `start_runner`, where the fork logic lives in https://github.com/ManageIQ/manageiq/pull/14256 (the PR containing 18ffca9)